### PR TITLE
Initial compile

### PR DESCRIPTION
**Pull request by drchrislewis**
A bunch of things to get it to compile.
A few syntax errors fixed.
Added a queue of images to account for delay in transforms being published.
disabled some features (resetting the volume parameters)
Now returns mesh in tsdf_frame instead of world. 
Needs a real cleanup, but does generate multiple scans.

### CODE REVIEW

**Comment by DavidMerzJr** [comment]
@drchrislewis Does this PR depend on your [pull request against yak](https://github.com/ros-industrial/yak/pull/8)?

**Comment by drchrislewis** [comment]
@DavidMerzJr Yes, just so the system is equivalent to the one on site. There are some things that are disabled that will need to be re-enabled.

File: yak_ros/src/yak_ros1_node.cpp [hunk 1]
```diff
@@ -62,43 +70,63 @@ class OnlineFusionServer
...
+      if(motion_mag < DEFAULT_MINIMUM_TRANSLATION)
+	{
+	  ROS_INFO_STREAM("Camera motion below threshold");
+	  return;
+	}
+      
+      cv_bridge::CvImagePtr cv_ptr = cv_bridge::toCvCopy(next_image, sensor_msgs::image_encodings::TYPE_16UC1);
+
+#if (0)// debug the transform by printing it out every 30'th update
+      static int q=0;
```
Comment (drchrislewis) [inline yak_ros/src/yak_ros1_node.cpp:106]: @DavidMerzJr right now, this only accepts the raw depth image which is a 16UC1. The rectified image is 32FC1 so it does not work. We need the rectified image for intrinsic calibration to have any affect.

File: yak_ros/src/yak_ros1_node.cpp [hunk 2]
```diff
@@ -160,7 +200,7 @@ class OnlineFusionServer
 int main(int argc, char **argv)
 {
   ros::init(argc, argv, "tsdf_node");
-  ros::NodeHandle nh("tsdf_node");
+  ros::NodeHandle nh; // removed "tsdf_node" from node handle to keep service names cleaner
```
Comment (drchrislewis) [inline yak_ros/src/yak_ros1_node.cpp:203]: @DavidMerzJr We should be getting the camera matrix from the camera_info_manager services of the camera driver, not from parameters set in the launch file.

**Pull request merged**